examples/text-classification/README.md

```diff
@@ -155,3 +155,59 @@ python run_glue.py \
   --use_hpu_graphs_for_inference \
   --bf16
 ```
+
+## Llama Guard for GLUE MRPC task
+
+Llama Guard can be used for text classification task. The *transformers* library will change the head of the model for you during fine-tuning or inference. You can use same general command as for BERT, except you need to add `--add_pad_token=True`, because Llama based models don't have `pad_token` in their model and tokenizer configuration files. So, `--add_pad_token=True` will add `pad_token` to tokenizer and model's config as `eos_token` if it's not defined.
+
+### Fine-tuning with DeepSpeed
+
+Llama Guard can be fine-tuned with DeepSpeed, here is how you would do it on the text classification MRPC task using DeepSpeed with 8 HPUs:
+
+```bash
+python ../gaudi_spawn.py /
+    --world_size 8 --use_deepspeed run_glue.py \
+    --model_name_or_path meta-llama/LlamaGuard-7b \
+    --gaudi_config gaudi_config.json  \
+    --task_name mrpc  \
+    --do_train  \
+    --do_eval \
+    --per_device_train_batch_size 32 \
+    --per_device_eval_batch_size 8  \
+    --learning_rate 3e-5  \
+    --num_train_epochs 3 \
+    --max_seq_length 128 \
+    --add_pad_token True \
+    --output_dir /tmp/mrpc_output/ \
+    --use_habana \
+    --use_lazy_mode \
+    --use_hpu_graphs_for_inference \
+    --throughput_warmup_steps 3  \
+    --deepspeed ../../tests/configs/deepspeed_zero_2.json
+```
+
+You can look at the [documentation](https://huggingface.co/docs/optimum/habana/usage_guides/deepspeed) for more information about how to use DeepSpeed in Optimum Habana.
+
+> If your model classification head dimensions do not fit the number of labels in the dataset, you can specify `--ignore_mismatched_sizes` to adapt it.
+
+### Inference
+
+You can run inference with Llama Guard on GLUE on 1 Gaudi card with the following command:
+
+```bash
+python run_glue.py \
+  --gaudi_config gaudi_config.json \
```

Comment:
where is the gaudi_config.json?

Reply:
@dsmertin  we just create https://huggingface.co/Habana/llama, can you try to use this?